source/type/pb_converter.h

```diff
@@ -34,6 +34,108 @@ www.navitia.io
 #include "type/response.pb.h"
 #include "type/pt_data.h"
 
+namespace pt = boost::posix_time;
+
+// we don't use a boolean to have a more type checks
+enum class DumpMessage {
+    Yes,
+    No
+};
+
+#define null_time_period boost::posix_time::time_period(boost::posix_time::not_a_date_time, boost::posix_time::seconds(0))
+
+//*********************************************
+
+using house_number_coord = std::pair<int, const navitia::type::GeographicalCoord&>;
+using way_name = std::pair<const navitia::georef::Way*, const std::string&>;
+using way_pair = std::pair<const navitia::georef::Way*, house_number_coord>;
+using way_pair_name = std::pair<way_name, house_number_coord>;
```

Comment:
Please, no pair of pair. If you really need something like that, create a structure with 3 fields with meaningful names.